Atomic/storage.py

```diff
@@ -111,8 +114,24 @@ class Storage(Atomic):
     dss_conf = "/etc/sysconfig/docker-storage-setup"
     dss_conf_bak = dss_conf + ".bkp"
 
+    def __init__(self):
+        super(Storage, self).__init__()
+
+        if self.args.graph:
+            self.graphdir = self.args.graph
+        else:
+            if os.path.exists("/var/lib/docker") and os.path.exists("/var/lib/docker-latest"):
+                raise ValueError("You must select the graph storage path to reset: /var/lib/docker or /var/lib/docker-latest")
+            if os.path.exists("/var/lib/docker"):
+                self.graphdir = "/var/lib/docker"
+            else:
+                if os.path.exists("/var/lib/docker-latest"):
+                    self.graphdir = "/var/lib/docker-latest"
+                else:
+                    raise ValueError("You must select the graph storage path to reset")
```

Comment:
We probably could be little more help and say something like, `Could not find any default graph storage path. Specify one using --graph option`